content/concepts/discovery-routing/pubsub.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "PubSub"
+description: "."
+weight: 225
+---
```

Comment:
I would move PubSub out of the discovery and routing section and instead place it a section like messaging. Pubsub can be employed for routing/discovery but it's more for message dissemination imo

Reply:
I intended to remove this (and didn't create another doc as a result), thanks!